create_infotexts.py

```diff
@@ -375,41 +369,28 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
                 place_as_wikitext = place_string
 
             # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
-            if places_mapping[place_string].get('commonscat') and \
-            places_mapping[place_string].get('commonscat') != "-" and \
-            places_mapping[place_string].get('commonscat') not in self.content_cats:
-
+            if places_mapping[place_string].get('commonscat'):
                 self.content_cats.append(places_mapping[place_string]["commonscat"])
 
         else:
-            self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
-            place_as_wikitext = place_string
+            for place in places_mapping:
+                if place.lower() in desc_string.lower():
 
-
-        for place in places_mapping:
-            if place.lower() in desc_string.lower():
-
-                if places_mapping[place]["wikidata"] and places_mapping[place]["wikidata"] != "-":
-                    if place_as_wikitext: # There is already a depicted place found in field <Ort, foto>
-                        place_as_wikitext += "/{{{{city|1={wikidata}}}}}".format(
-                            wikidata=places_mapping[place]["wikidata"]
-                            )
-                    else:
+                    if places_mapping[place]["wikidata"]:
                         place_as_wikitext = "{{{{city|1={wikidata}}}}}".format(
```

Comment:
Every time we get here (remember that we are looping) we will be overwriting the last value. Same is true for `place_as_wikitext = place`. I guess that instead we want to maintain a list of matches which we join in the end?

See also my [previous question](https://github.com/mattiasostmar/SMVK-Cypern_2017-01/pull/30#discussion_r110140513) is two matches actually a reason to stop not add any of them (and just add not mapped meta-cat)?